http-clients/src/main/java/com/palantir/remoting3/clients/UserAgent.java

```diff
@@ -34,18 +31,15 @@
 @ImmutablesStyle
 public interface UserAgent {
 
-    List<Agent> agents();
+    String DEFAULT_VERSION = "0.0.0";
```

Comment:
`unknown`?

Reply:
need to special-case parsing then. can do.

Reply:
0.0.0 is probably fine tbh

Reply:
because it doesn't match the expected regex?

Reply:
we're putting a string here to mean "the version is unknown", the word "unknown" and the string "0.0.0" are about equivalent

Reply:
back to 0.0.0